mod/wikirate_assets/spec/set/type/image_spec.rb

```diff
@@ -1,27 +1,19 @@
 RSpec.describe Card::Set::Type::Image do
   describe "missing view" do
-    before do
-      @file = File.open("#{Rails.root}/mod/wikirate_assets/spec/set/right/image.jpg")
-      @missing_card = Card["missing image"]
+    def new_image_view view
+      Card.new(type_id: Card::ImageID, name: "TestImage").format.render view
     end
 
-    it "shows missing view because of denied" do
-      image_card = Card.create! name: "TestImage", type_id: Card::ImageID,
-                                image: @file
-      missing_image = image_card.format.subformat(@missing_card)._render_core
-
-      image_card_format = image_card.format
-      image_card_format.instance_variable_set "@denied_view", :core
+    it "is NOT wrapped when rendered from core" do
+      expect(new_image_view(:core)).not_to match(/missing-view/)
+    end
 
-      expect(image_card_format.render_missing).to eq(missing_image)
```

Comment:
This really seemed like a lot of hullaballoo for testing very little.  Am I missing something?

Reply:
probably very old